packages/e2e-tests/specs/editor/blocks/__snapshots__/navigation.test.js.snap

```diff
@@ -6,6 +6,8 @@ exports[`Navigation allows an empty navigation block to be created and manually
 <!-- wp:navigation-link {\\"label\\":\\"Contact\\",\\"type\\":\\"page\\",\\"id\\":[number],\\"url\\":\\"https://this/is/a/test/search/get-in-touch\\",\\"kind\\":\\"post-type\\",\\"isTopLevelLink\\":true} /-->"
 `;
 
+exports[`Navigation allows pages to be created from the navigation block and their links added to menu 1`] = `""`;
```

Comment:
This is wrong, there should be a page list. It's working in #37358, so once that's merged I'll bring across those changes.